lib/Scheduling/SimplexSchedulers.cpp

```diff
@@ -261,6 +261,7 @@ class ModuloSimplexScheduler : public CyclicSimplexScheduler {
   enum { OBJ_LATENCY = 0, OBJ_AXAP /* i.e. either ASAP or ALAP */ };
   bool fillObjectiveRow(SmallVector<int> &row, unsigned obj) override;
   void updateMargins();
+  unsigned int resII();
```

Comment:
Nits:
- Better name: `computeResMinII`
- Just use `unsigned`
- Place declaration below `scheduleOperation` (to have the same order as the method implementations)